frontend/packages/kubevirt-plugin/src/components/modals/save-and-restart-modal/save-and-restart-modal.ts

```diff
@@ -7,8 +7,10 @@ import * as _ from 'lodash';
 export const saveAndRestartModal = (vm: VMKind, vmi: VMIKind, saveChanges?: () => void) =>
   confirmVMIModal({
     vmi,
-    title: 'Restart Virtual Machine',
-    alertTitle: 'Restart Virtual Machine alert',
+    // t('kubevirt-plugin~Restart Virtual Machine')
+    title: 'kubevirt-plugin~Restart Virtual Machine',
```

Comment:
`ConfirmVMIModal` is not using `t` to translate these keys

Reply:
Fixed!